### PR TITLE
service_facts: return service state information on AIX

### PR DESCRIPTION
**Pull request by dberg1**
##### SUMMARY

AIX uses the System Resource Controller (SRC) to manage services.
See https://www.ibm.com/support/knowledgecenter/ssw_aix_72/osmanagement/sysrescon.htm
Use lssrc command on AIX to return service state information.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

service_facts

##### ADDITIONAL INFORMATION

Before:
```
ok: [aixhost1] => {
    "ansible_facts.services": "VARIABLE IS NOT DEFINED!: 'dict object' has no attribute 'services'"
}
```

After:
```
ok: [aixhost1] => {
    "ansible_facts.services": {
        "IBM.AuditRM": {
            "name": "IBM.AuditRM",
            "source": "src",
            "state": "stopped"
        },
        "IBM.ConfigRM": {
            "name": "IBM.ConfigRM",
            "source": "src",
            "state": "running"
        },
        "IBM.DRM": {
            "name": "IBM.DRM",
            "source": "src",
            "state": "running"
        },
        "IBM.ERRM": {
            "name": "IBM.ERRM",
            "source": "src",
            "state": "stopped"
        },
        "IBM.HostRM": {
            "name": "IBM.HostRM",
            "source": "src",
            "state": "running"
        },
        "IBM.LPRM": {
            "name": "IBM.LPRM",
            "source": "src",
            "state": "stopped"
        },
        "IBM.MgmtDomainRM": {
            "name": "IBM.MgmtDomainRM",
            "source": "src",
            "state": "running"
        }
    }
}
```

### CODE REVIEW

File: lib/ansible/modules/service_facts.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 description:
      - Return service state information as fact data for various service management utilities
 version_added: "2.5"
-requirements: ["Any of the following supported init systems: systemd, sysv, upstart"]
+requirements: ["Any of the following supported init systems: systemd, sysv, upstart, AIX SRC"]
```
Comment (bcoca) [inline lib/ansible/modules/service_facts.py:19]: add note that AIX SRC was added in 2.11

**Comment by relrod** [comment]
Can you add a `minor_changes` [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) for the change?

**Review by samdoran — CHANGES_REQUESTED** [review]
It would also be good to have unit tests with the output of `lssrc` as a fixture. We don't currently have AIX available for integration testing.

File: lib/ansible/modules/service_facts.py (hunk 1 above)
Comment (dberg1) [reply lib/ansible/modules/service_facts.py:19]: Done

**Comment by dberg1** [comment]
I have added the requested unit test with the output of `lssrc`. Please review.

**Pull request merged**